ascify art/Ascify-Art.py

```diff
@@ -119,137 +119,140 @@ def getChar(inputInt):
     pix = im.load()
     outputImage = Image.new('RGB', (oneCharWidth * width, oneCharHeight * height), color = background)
     d = ImageDraw.Draw(outputImage)
-    
+
     new_r = int(slider_r.get())
     new_g = int(slider_g.get())
     new_b = int(slider_b.get())
     auto = automatic.get()
-    
+
     # replace the pixels with text
     for i in range(height):
         for j in range(width):
             r, g, b = pix[j, i]
-            
+
             if auto==0:
                 if r>=new_r: r = new_r
                 if g>=new_g: g = new_g
                 if b>=new_b: b = new_b
-            
+
             h = int(r/3 + g/3 + b/3)
             pix[j, i] = (h, h, h)
             d.text((j*oneCharWidth, i*oneCharHeight), getChar(h), font = fnt, fill = (r, g, b))
-                
-                
-    # some other enhancements like saturation and brightness            
+
+
+    # some other enhancements like saturation and brightness
     outputImage = ImageEnhance.Color(outputImage).enhance(slider_sat.get())
     outputImage = ImageEnhance.Brightness(outputImage).enhance(slider_br.get())
 
     # update the label
     image = customtkinter.CTkImage(outputImage, size=(frame_left.winfo_height(),frame_left.winfo_height()*img.size[1]/img.size[0]))
     label_image.configure(image=image)
-    
+
 def openfile():
     # opening and loading the file
     global file, image, img, dir_, sequence, previous
-    file = filedialog.askopenfilename(filetypes =[('Images', ['*.png','*.jpg','*.jpeg','*.bmp','*webp']),('All Files', '*.*')])
-    if file:
+    if file := filedialog.askopenfilename(
+        filetypes=[
+            ('Images', ['*.png', '*.jpg', '*.jpeg', '*.bmp', '*webp']),
+            ('All Files', '*.*'),
+        ]
+    ):

```

Comment:
Function `openfile` refactored with the following changes:

- Use named expression to simplify assignment and conditional
- Use f-string instead of string concatenation [x5]
- Merge else clause's nested if statement into elif